CHANGELOG.rst

```diff
@@ -8,6 +8,11 @@ Release Notes
 Prelude
 -------
 
+Please note that a critical bug has been identified in this release that would prevent the kubernetes integration from collecting kublet metrics.
+The severity of the issue has led us to remove the packages for the
+affected platform (**Docker**) and to make the `latest` tag point to the `6.7.0` release. If you have upgraded to this version,
+on **Docker** we recommend you downgrade to ``6.7.0``.
```

Comment:
`on **Docker** or **Kubernetes**` I know Docker is the platform we target, but this applies even if people use k8s with a different runtime than docker.

Reply:
There are people that use the linux packages (and probably install with a package manager) but still enable the k8s integration right ? Would `If you have upgraded to this version of the containerized agent we recommend you downgrade to ``6.7.0``.` be less confusing ?

Reply:
yeah that works